Package.swift

```diff
@@ -35,6 +37,10 @@ let package = Package(
         .target(name: "RevenueCat",
                 path: "Sources",
                 exclude: ["Info.plist", "LocalReceiptParsing/ReceiptParser-only-files"]),
+        .target(name: "RevenueCat_CustomEntitlementComputation",
+                path: "CustomEntitlementComputation",
+                exclude: ["Info.plist", "LocalReceiptParsing/ReceiptParser-only-files"],
+                swiftSettings: [.define("ENABLE_CUSTOM_ENTITLEMENT_COMPUTATION")]),
```

Comment:
Unrelated to this PR and no need to fix it here, but mentioning before I forget again:
I noticed this today when playing around with a solution: 
this file is only meant for receiptParser, but it's not in the folder. It is only marked for target membership in ReceiptParser, though, so we have a disconnect there
<img width="407" alt="image" src="https://user-images.githubusercontent.com/3922667/234386515-81a562b9-5c86-4efe-bd22-f5065accced3.png">

Reply:
Hmm yeah we should move that.